src/engraving/libmscore/box.cpp

```diff
@@ -58,8 +58,8 @@ static const ElementStyle hBoxStyle {
 //   Box
 //---------------------------------------------------------
 
-Box::Box(Score* score)
-    : MeasureBase(score)
+Box::Box(const ElementType& type, Score* score)
```

Comment:
is there a reason to bother passing an enum as a reference? (other than the possibility of it being changed to an expensive-to-copy type in the future, but that seems unlikely...)

Reply:
there is no particular reason, just a habit, I think this is not good and not bad.